### PR TITLE
updating the readme file from my New branch

### PR DESCRIPTION
**Pull request by Parvathimenon1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 -	Calculating the area of rectangle using different method program
```
Comment (sunishsurendrank) [inline README.md]: Remove extra space

Comment (sunishsurendrank) [inline README.md]: Some points starts with Capital letter some in small letter, please make sure it is same all the place

Comment (Parvathimenon1) [reply README.md]: done

Comment (Parvathimenon1) [reply README.md]: Done

**Pull request merged**